src/berry_crush.c

```diff
@@ -77,37 +87,22 @@ struct BerryCrushGame_5C
     u8 unk02_1:1;
     u8 unk02_2:1;
     u8 unk02_3:5;
-    u8 unk03;
+    s8 unk03;
     u16 unk04;
     u16 unk06;
     u16 unk08;
     u16 unk0A;
 };
 
-union BerryCrushGame_68
+struct BerryCrushGame_68
 {
-    struct BerryCrushGame_68_x
-    {
-        struct BerryCrushGame_68_x_SubStruct
-        {
-            s32 unk00;
-            u16 unk04;
-            s16 unk06;
-            u16 unk08;
-            u16 unk0A;
-            u16 unk0C[2][5];
-            u8 filler20[16];
-        } unk00;
-        u8 unk30[12];
-        struct BerryCrushGame_Player others[4];
-        u8 fillerBC[20];
-    } as_four_players;
-    struct BerryCrushGame_68_y
-    {
-        u8 filler00[28];
-        struct BerryCrushGame_Player unk1C[5];
-        u8 fillerBC[20];
-    } as_five_players;
+    u32 unk00;
+    u16 unk04;
+    u16 unk06;
+    u16 unk08;
+    u16 unk0A;
+    u16 unk0C[2][5];
+    u8 filler20[2][8];
```

Comment:
This is in use, so it should be `unk` not `filler`.

Reply:
done